document/core/exec/numerics.rst

```diff
@@ -135,6 +152,143 @@ When a number is stored into :ref:`memory <syntax-mem>`, it is converted into a
 Again these functions are invertable bijections.
 
 
+.. index:: SIMD, shape
+.. _aux-simdto:
+.. _aux-simdof:
+
+SIMD
+....
+
+SIMD values have the same underlying representation as an |i128|. They can also be interpreted as numeric values packed into a |V128| with a particular |shape|.
+
+.. math::
+   \begin{array}{lll@{\qquad}l}
+   \simdto_{t\K{x}N}(c) &=&
```

Comment:
3. This is the key aux function used for almost all simd operations, it converts a i128 into a sequence of numeric values. This is generalized over all shapes (doesn't care about i32x4 or f32x4, they are both converted to a sequence of length 4).

Reply:
we already have `shape` defined, it's a non-terminal that represents "i32x4" and friends.
Would calling this function `shape` as well be confusing?
Maybe we can call it `to_shape` or something.

Reply:
Hm, yeah, it might be a bit odd. In math you don't usually have "to" prefixes. How about "shaped", somewhat analogous to "unpacked"?

Reply:
Yea shaped works, `shaped_{i64x2}(c)`, though I have to think it for half a second to see what it does - the grammar doesn't look like something I'm used to (this is more declarative than verb-y)

Couple more suggestions we might want to consider (in no order of preference)
-  `elements_{i64x2}(c)`
-  `scalarize_{i64x2}(c)`
-  `split_{i64x2}(c)`
-  `sequence_{i64x2}(c)`
-  `lanes_{i64x2}(c)` (this is a collision with an existing definition, which is an auxiliary function that returns the number of lanes, but we can rename that to num_lanes)

Reply:
@rossberg what do you think about the list of suggestion, does any one look better than the rest?

Reply:
@rossberg any thoughts? Do you still prefer `shaped`?

Reply:
Ah, sorry, I somehow had missed these. `lanes` actually sounds best to me. How about we rename the existing `lanes` function to `dim`?

Reply:
Sg, renamed `lanes` to `dim`, then renamed `simdto` to `lanes`, in two separate commit, hopefully easier to review.